circuit/src/androidMain/kotlin/com/slack/circuit/Navigator.android.kt

```diff
@@ -16,19 +15,34 @@ import com.slack.circuit.backstack.SaveableBackStack
  *   back handler. Defaults to true.
  * @see NavigableCircuitContent
  */
+@Deprecated(
```

Comment:
I was unsure whether to deprecate or just remove this API, since due to the back handling being handled by default in `NavigableCircuitContent` (for reals this time) it's no longer necessary.